Announce Keptn 0.16.0 (#1246)
keptn/keptn.github.io: content/docs/0.16.x/operate/upgrade/index.md

## Upgrade from Keptn 0.15.x to Keptn 0.16.x

> With Keptn 0.16.x, every Keptn project requires an upstream. Hence, before upgrading Keptn it is required to (i) attach an upstream to your Keptn projects and (ii) do a [backup](..//backup_and_restore/#back-up-configuration-service).

Reviewer: ```suggestion
With Keptn 0.16.x, every Keptn project requires an [upstream Git repository](../manage/git_upstream). Hence, before upgrading Keptn. you must (i) attach an upstream to your Keptn projects and (ii) do a [backup](..//backup_and_restore/#back-up-configuration-service).
```

> If you set an upstream for all your Keptn projects, no additional steps are required.

Reviewer: ```suggestion
If you already have an upstream for all your Keptn projects, no additional steps are required.
```

Suppose you need the additional features provided by the *resource-service*,  such as HTTPS/SSH or Proxy, to configure your Keptn project with an upstream. In that case,
you can also deploy the *resource-service* and configure the Git repositories later. For this, a backup is necessary.

> 1. Back up of the [configuration-service](../backup_and_restore/#back-up-configuration-service).

Reviewer: ```suggestion
1. Back up the [configuration-service](../backup_and_restore/#back-up-configuration-service).
```
